src/auth.ts

```diff
@@ -129,9 +133,12 @@ Auth.login = (req: Request, res: Response, next: NextFunction) => {
 };
 
 Auth.logout = (req: Request, res: Response) => {
-  req.logout();
-  req.session.destroy(err => console.error(`Failed to destroy the session: ${err}`)); // eslint-disable-line no-console
-  res.redirect(samlLogout);
+  if (!req.user) res.redirect('/');
+  return Auth.passportStrategy.logout(req, (error, uri) => {
+    req.session.destroy(err => console.error(`Failed to destroy the session: ${err}`)); // eslint-disable-lin no-console
```

Comment:
it looks like `esline-disable-lin ...` is missing the `e` in `line`.. I wonder how it is not failing to do what it intends on doing?! :D

Reply:
Pushed a new commit to fix this